security.md

```diff
@@ -4,17 +4,32 @@ title: "Security Policy"
 
 Security threats and vulnerabilities affect everyone using R-multiverse. Issues may include (but are not limited to):
 
+* Malware in R packages contributed to R-multiverse.
```

Comment:
```suggestion
* Malware or exploitable code in R packages contributed to R-multiverse.
```

Reply:
Just didn't want to leave the impression that we're somehow concerned about malware specifically. As we discussed, we're not more vulnerable to this risk than other repositories, and in fact in the first instance we're just a listing with the content hosted on GitHub/GitLab.